pkg/apis/extensions/v1alpha1/types_worker.go

```diff
@@ -277,6 +277,5 @@ type MachineDeployment struct {
 	// Maximum is the maximum number for this machine deployment.
 	Maximum int32 `json:"maximum"`
 	// Priority (or weight) is the importance by which this machine deployment will be scaled by cluster autoscaling.
-	// +optional
-	Priority *int32 `json:"priority,omitempty"`
+	Priority int32 `json:"priority,omitempty"`
```

Comment:
Leaving `omitempty` here still leaves the field as optional in the generated worker CRD spec. Is this intended? 
Note that this could be confusing since the doc in `docs/api-reference/extensions.md` was updated to drop the `<em>(Optional)</em>` description.